api/src/functions/auth.js

```diff
@@ -27,12 +27,15 @@ export const handler = async (event, context) => {
     // were included in the object given to the `signUp()` function you got
     // from `useAuth()`
     signupHandler: ({ username, hashedPassword, salt, userAttributes }) => {
+      if (!username.match(/.+?@.+?\..+?/)) {
+        throw new Error('Not a valid email address!')
+      }
```

Comment:
require email on sign up for username (as we dont have username but rather email)